tests/integration/components/ui-table/cell/events/view/videoroom/cell-stream-url-test.ts

```diff
@@ -7,11 +7,30 @@ module('Integration | Component | ui-table/cell/events/view/videoroom/cell-strea
   setupRenderingTest(hooks);
 
   test('it renders', async function(assert) {
-    this.set('record', { id: 23, name: 'Main Hall', slugName: 'main-hall' });
-    this.set('extraRecords', { event: { identifier: 'abcd' } });
+    this.set('record', { get(dataType: string) {
```

Comment:
Instead, you can use EmberObject.create

Reply:
its showing same error if I use EmberObject.create